packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/Chatbot.md

```diff
@@ -59,13 +60,13 @@ This demo displays a basic chatbot, which includes:
 2. A [`<ChatbotHeader>`](/patternfly-ai/chatbot/chatbot-header) with all built sub-components laid out, including a `<ChatbotHeaderTitle>` that changes its presentation depending on the display mode.
```

Comment:
We want to adjust the link in line 59 to be  /patternfly-ai/chatbot/chatbot-container if we're renaming it.